package.json

```diff
@@ -67,5 +68,8 @@
     "testMatch": [
       "**/test/provider/**/*.(ts|tsx)"
     ]
+  },
+  "precommit": {
+    "run": []
```

Comment:
Can you add `check` here so that this is ready to go - we'll just hold off on merging until #16 is in.